src/ECER.Clients.PSPPortal/ecer.clients.pspportal.client/src/components/pages/Dashboard.vue

```diff
@@ -213,6 +229,14 @@ export default defineComponent({
 
   },
   methods: {
+    async getInstitutionData() {
+      this.educationInstitution = await getEducationInstitution();
+    },
+    async getProgramProfileData() {
+      let status: ProgramStatus = "Draft";
+      let statuses: ProgramStatus[] = new Array(status);
+      this.programsRequiringReview = await getPrograms(statuses);  
```

Comment:
I think you can compress this to. 

this.programsRequiringReview = await getPrograms(["Draft"]);

or 
this.programsRequiringReview = await getPrograms([ProgramStatus.Draft]);

typescript will check to make sure we're putting in the correct value.